local-cli/rnpm/windows/index.js

```diff
@@ -1,33 +1,6 @@
+const minimistConfig = require('./src/config');
+
 module.exports = [
-  {
```

Comment:
Is it some kind of best practice to have a very minimal index.js? Otherwise, why move the configs to config.js?

Reply:
I changed my mind on why I originally did this, I will revert.